roles/swift-common/defaults/main.yml

```diff
@@ -109,4 +125,5 @@ swift:
         no_replicators: 2
         no_auditors: 1
         no_updaters: 1
-  venv_dir: '/opt/openstack/current/swift'
+  venv_dir: '/opt/openstack/base'
```

Comment:
this will break ubuntu upgrade path (probably?).
ubuntu and centos (given is install from source) can still use venv_dir: '/opt/openstack/current/swift' like is it.
for rhel we should just do  pip install at system level as swfittool relies on other swift pkgs.